lua/lvim/lsp/providers/sumneko_lua.lua

```diff
@@ -7,13 +7,27 @@ local opts = {
       workspace = {
         library = {
           [require("lvim.utils").join_paths(get_runtime_dir(), "lvim", "lua")] = true,
-          [vim.fn.expand "$VIMRUNTIME/lua"] = true,
-          [vim.fn.expand "$VIMRUNTIME/lua/vim/lsp"] = true,
         },
         maxPreload = 100000,
         preloadFileSize = 10000,
       },
     },
   },
 }
-return opts
+
+local lua_dev_loaded, lua_dev = pcall(require, "lua-dev")
+if not lua_dev_loaded then
+  return opts
+end
+
+local dev_opts = vim.tbl_deep_extend("force", {
+  library = {
+    vimruntime = true, -- runtime path
+    types = true, -- full signature, docs and completion of vim.api, vim.treesitter, vim.lsp and others
+    -- plugins = true, -- installed opt or start plugins in packpath
+    -- you can also specify the list of plugins to make available as a workspace library
+    plugins = { "plenary.nvim" },
+  },
+}, { lspconfig = opts })
```

Comment:
Sorry! I forgot that we can pass the other opts like this, so there's no need for merging them.
```suggestion
local dev_opts =  {
  library = {
    vimruntime = true, -- runtime path
    types = true, -- full signature, docs and completion of vim.api, vim.treesitter, vim.lsp and others
    -- plugins = true, -- installed opt or start plugins in packpath
    -- you can also specify the list of plugins to make available as a workspace library
    plugins = { "plenary.nvim" },
  },
  lspconfig = opts
}
```

Reply:
done